src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs

```diff
@@ -158,7 +159,7 @@ internal partial class InProcLanguageServer : IAsyncDisposable
             var service = serviceContainer.GetFullAccessServiceBroker();
 
             var configuration = await TraceConfiguration.CreateTraceConfigurationInstanceAsync(service, cancellationToken).ConfigureAwait(false);
-            var traceSource = await configuration.RegisterLogSourceAsync(logId, new LogHub.LoggerOptions(), cancellationToken).ConfigureAwait(false);
+            var traceSource = await configuration.RegisterLogSourceAsync(logId, new RpcContracts.Logging.LoggerOptions(), cancellationToken).ConfigureAwait(false);
```

Comment:
There's some LogHub related change. @dibarbet would you be the right person to review that?

Reply:
I think this has been taken care of in main-vs-deps - you can just pull in whatever is there for this
https://github.com/dotnet/roslyn/blob/main-vs-deps/src/VisualStudio/Core/Def/Implementation/LanguageClient/InProcLanguageServer.cs#L177

Reply:
I don't want to pull in more changes than necessary for this branch. Does this change here looks ok to you, so we can delay sync until interop updates are done?

Reply:
Yeah this seems fine until we merge 16.10 in here.